nservicebus/operations/metrics.md

```diff
@@ -7,8 +7,7 @@ related:
  - samples/metrics
 ---
 
-Metrics collects information to measure endpoint health and performance.
-
+Metrics collects information to measure endpoint health and performance. When a system is broken down into multiple processes, each with its own queue, it allows identifying which process is the bottleneck by examining how many messages (on average) are in each queue. The only issue is that without knowing the rate of messages coming into each queue, and the rate at which messages are being processed from each queue, it is not possible to know how long messages are waiting in each queue, which is the primary indicator of a bottleneck.
```

Comment:
> Metrics collects information

`NServiceBus.Metrics` collects information? Or if we want to talk about metrics in general then we need to rephrase the first sentence.